### PR TITLE
fix(cache-control): Extend cache control to s-maxage 900, s-w-r 1000 :clubs:

### PR DESCRIPTION
**Pull request by Jeevan-Kishore**
In an effort to increase cache hits,

Changing `s-maxage 900`, `stale-while-revalidate 1000`

- [x] Isomorphic routes
- [x] Visual stories
- [x] Reroute

### CODE REVIEW

File: server/handlers/cdn-caching.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ exports.addCacheHeadersToResult = function addCacheHeadersToResult(res, cacheKey
       res.setHeader('Vary', "Accept-Encoding");
       res.setHeader('Surrogate-Control', "private,no-cache,no-store,max-age=0");
     } else {
-      res.setHeader('Cache-Control', "public,max-age=15,s-maxage=240,stale-while-revalidate=300,stale-if-error=14400");
+      res.setHeader('Cache-Control', "public,max-age=15,s-maxage=900,stale-while-revalidate=1000,stale-if-error=14400");
```
Comment (chchitti) [inline server/handlers/cdn-caching.js]: I think you should do this change on lines 17, 22 and 24 also.

Comment (Jeevan-Kishore) [reply server/handlers/cdn-caching.js]: wasn't sure on whether we should be having this change for requests without `cachekeys`.

Comment (sinujohn91) [reply server/handlers/cdn-caching.js]: isn't 900 a bit too high?

Comment (sinujohn91) [reply server/handlers/cdn-caching.js]: else we should enable black knight to purge cache whenever a new deploy happens.

Comment (Jeevan-Kishore) [reply server/handlers/cdn-caching.js]: @tribu has recommended the values above. BK to purge is not a bad idea, circleci if possible would be better though. 

Should we proceed merging this? 

@gja @sinujohn91 ^^^

Comment (Jeevan-Kishore) [reply server/handlers/cdn-caching.js]: Approved. Merging to master, resolving conversation.

**Pull request merged**